guides/v2.1/contributor-guide/contributing_dod.md

```diff
@@ -166,6 +166,20 @@ The reviewer should check whether the task meets the original acceptance criteri
 * All changes are documented properly
 * All changes are covered with automated tests
 * Determine if code changes caused any failure in continuous integration builds
+* All backward-incompatible changes on code level are approved by an architect. The following cases currently cause SVC failures but do NOT require architect approval:
```

Comment:
Would be good to move it closer to "There are no unauthorized backward-incompatible" item and maybe phrase those similarly.

Reply:
It's better at first to improve SVC tool and do not trigger failures for such cases.

Reply:
@buskamuza done